--- expat-2.1.1/configure.ac~	2012-07-28 10:41:03.000000000 +0300
+++ expat-2.1.1/configure.ac	2012-07-28 10:41:41.511402972 +0300
@@ -51,8 +51,6 @@
 
 AC_CONFIG_HEADER(expat_config.h)
 
-sinclude(conftools/ac_c_bigendian_cross.m4)
-
 AC_LIBTOOL_WIN32_DLL
 AC_PROG_LIBTOOL
 
@@ -83,9 +80,7 @@
 
 dnl Checks for typedefs, structures, and compiler characteristics.
 
-dnl Note: Avoid using AC_C_BIGENDIAN because it does not
-dnl work in a cross compile.
-AC_C_BIGENDIAN_CROSS
+AC_C_BIGENDIAN
 
 AC_C_CONST
 AC_TYPE_SIZE_T
